alt/issues.md

```diff
@@ -0,0 +1 @@
+[ALT's bugzilla page](https://bugzilla.altlinux.org/enter_bug.cgi?product=Sisyphus) (choose `docker-ce` as component and include details about image problems in the description) or [GitHub](%%GITHUB-REPO%%/issues)
```

Comment:
Do you think it would make sense to link to https://bugzilla.altlinux.org/buglist.cgi?query_format=advanced&product=Sisyphus&component=docker-ce so folks can browse existing issues before filing a new one?

Reply:
Yes. Thank you.